kill_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+)
+
+func TestProcessSignal(t *testing.T) {
```

Comment:
Or another way is to actually create each test such as `TestProcessSignalInvalidSignal()`, `TestProcessSignalInvalidShortSignal()`, ... into a generic function like `testProcessSignalFailure()` or `testProcessSignalSuccess()`.
That way each unit test has a name and it is very quick to get which test failed.